tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -26,10 +27,16 @@ ${MS_TABLE_STATUS_FAILURE}=    //span[contains(@class,"pf-c-icon__content")][con
 *** Keywords ***
 Open Model Serving Home Page
     [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
-    Page Should Contain    Model Serving
-    Click Link      Model Serving
-    Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
-    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    SeleniumLibrary.Page Should Contain    Model Serving
+    SeleniumLibrary.Click Link      Model Serving
+    Sleep    1s
```

Comment:
could you replace the sleep with `Wait for RHODS Dashboard to Load`?

Reply:
Wouldn't that keyword only wait for Model Serving to appear?